From 0e1950f5b9a930d4ac37af864ccc4359de044062 Mon Sep 17 00:00:00 2001
From: Dave Taht <dave.taht@bufferbloat.net>
Date: Thu, 8 Mar 2012 12:43:18 -0800
Subject: [PATCH] Compile small and include more top dir

Compiling for debugging is fine on a non-embedded system
There was also a bug that including .. should fix

SyncBase.h:31:31: fatal error: ccnr/ccnr_private.h: No such file or directory
---
 csrc/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/csrc/Makefile b/csrc/Makefile
index 88a4f24..5a81aac 100644
--- a/csrc/Makefile
+++ b/csrc/Makefile
@@ -15,7 +15,7 @@
 SUBDIRS = lib ccnd sync ccnr libexec cmd util tests exttests
 OBJTREE = build
 MAKEF =  -f ../conf.mk -f dir.mk -f ../subr.mk
-CFLAGS = -g
+CFLAGS = -Os -I..
 
 default all clean depend coverage test check shared install uninstall config_subdir: conf.mk subr.mk generic.mk _always
 	test -d include/ccn || (test -d ../include/ccn && mkdir -p include && ln -s ../../include/ccn include/ccn)
-- 
1.7.5.4

